src/utils/i18n.ts

```diff
@@ -21,9 +21,10 @@ i18next
           provinceOrTerritory: "Province or Territory",
           capitalCity: "Capital City",
           attempts: "Attempts",
-          gamePotRoundInstruction: "TODO", // TODO: we probably don't need this
-          gameFlagRoundInstruction: "Select the flag",
-          gameCapitalRoundInstruction: "What is the capital",
+          gamePotRoundInstruction: "Select the Province of",
+          gameFlagRoundInstruction: "Select the Flag of",
```

Comment:
1. _of_ shouldn't be in the instructions as that comes from the `geo` namespace in `i18n.ts` (due to French)
2. I would stick to spelling this as a sentence, meaning we only capitalize the first letter of the first word
3. `en`, `fr`, and `LOLcalize` don't have same keys - no French instructions for neighbors round

Reply:
got it